app/src/lib/stores/app-store.ts

```diff
@@ -554,6 +556,8 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
     if (__WIN32__) {
       const useWindowsOpenSSH = getBoolean(UseWindowsOpenSSHKey)
+      // TODO: is it ok to use a string literal here?
```

Comment:
Looking at the implementation of `askForConfirmationOnRepositoryRemoval` may be helpful reference as it is a similar setting. 

Preferably, 
- we set a key as constant at the top of file, see the `confirmRepoRemovalKey` as the example for the `askForConfirmationOnRepositoryRemoval` prompt setting.
- Additionally, as the location of this initialization is not quite ideal especially given this is in a Windows only if block and we would like applied to all operating systems. We typically initialize these in the `loadInitialState()` method. You can see `askForConfirmationOnRepositoryRemoval` on line 2095.